lib/AST/ASTPrinter.cpp

```diff
@@ -5426,6 +5625,7 @@ void PrintAST::visitCovariantReturnConversionExpr(CovariantReturnConversionExpr
 }
 
 void PrintAST::visitInterpolatedStringLiteralExpr(InterpolatedStringLiteralExpr *expr) {
+  visit(expr->getInterpolationExpr());
```

Comment:
As mentioned previously, implementing this will require traversing the TapExpr to reconstruct the string interpolation syntax.